### PR TITLE
rhc-13237: log host info when identity not provided

### PR DESCRIPTION
**Pull request by thearifismail**
## Overview

This PR is being created to address [this Jira](https://issues.redhat.com/browse/RHCLOUD-13237).
logging host info should help locating the source reporter with problematic message.

## Secure Coding Practices Checklist GitHub Link

- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: utils/payloads.py [hunk 1]
```diff
@@ -9,13 +9,12 @@
 # good
 IDENTITY = {
     "account_number": "test",
-    "type": "system",
+    "type": "System",
```
Comment (ryandillinfelton) [inline utils/payloads.py:12]: Is this case change important?

**Review by ryandillinfelton — COMMENTED** [review]
Looks like it does what it's supposed to but I had a question about the payloads change

File: utils/payloads.py (hunk 1 above)
Comment (thearifismail) [reply utils/payloads.py:12]: yes.  that's what is sent by the reporters, i.e with capital S.  I would rather stay with what reporters provide then be flexible and introduce a potential for drift.

Comment (ryandillinfelton) [reply utils/payloads.py:12]: Cool! That makes sense to me

**Review by kruai — APPROVED** [review]

**Pull request merged**